.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+name: Gradle Build Project Trigger
+
+on:
+  pull_request:
+    branches: [dev, main]
+    paths-ignore:
+      - '**.gradle/wrapper'
+      - '**.gitignore'
+      - 'LICENSE'
+      - 'THIRD PARTY NOTICES'
+      - '**.md'
+  workflow_dispatch:
```

Comment:
```suggestion
  workflow_dispatch:
  push:
```

Reply:
yup, adding the `push` with the correct exclusions 👍